src/main/java/com/spotify/docker/client/DefaultDockerClient.java

```diff
@@ -731,6 +731,34 @@ public ContainerCreation commitContainer(final String containerId,
   }
 
   @Override
+  public void renameContainer(final String containerId, final String name)
+          throws DockerException, InterruptedException {
+    WebTarget resource = resource()
+            .path("containers").path(containerId).path("rename");
+
+    if (name != null) {
```

Comment:
Assuming that the remote API requires a name to rename the container to, I think it is best to check for null and throw IllegalArgumentException.